package.json

```diff
@@ -78,6 +78,7 @@
     "sinon": "^1.15.4",
     "sinon-chai": "^2.8.0",
     "transfer-webpack-plugin": "^0.1.4",
+    "warning": "^2.1.0",
```

Comment:
I put this dependency in `devDependencies` since none of the code in this package should end up in a production build. Exactly because of the `process.env.NODE_ENV !== 'production'`.
Let me know if you still want to move it and I will.